demo/admin/src/pages/EditPage.tsx

```diff
@@ -166,6 +185,22 @@ export const EditPage: React.FC<Props> = ({ id, category }) => {
                             ),
                             content: rootBlocksApi.seo.adminUI,
                         },
+                        // {
+                        //     key: "dependencies",
+                        //     label: (
+                        //         <AdminTabLabel isValid={rootBlocksApi.seo.isValid}>
+                        //             <FormattedMessage id="pages.pages.page.edit.dependencies" defaultMessage="Dependencies" />
+                        //         </AdminTabLabel>
+                        //     ),
+                        //     content: (
+                        //         <DependencyList
+                        //             query={pageDependenciesQuery}
+                        //             variables={{
+                        //                 id: pageState?.document?.id ?? "",
+                        //             }}
+                        //         />
+                        //     ),
+                        // },
```

Comment:
This shouldn't be in an actual project, but I would like to keep it for testing `dependencies`